formula/src/main/java/com/instacart/formula/FormulaRuntime.kt

```diff
@@ -81,7 +81,7 @@ class FormulaRuntime<Input : Any, State, RenderModel : Any>(
                         threadChecker.check("Only thread that created it can trigger transitions.")
 
                         if (effects != null) {
-                            effectQueue.push(effects)
+                            effectQueue.addLast(effects)
```

Comment:
This is the fix: `push` appends the effect to the start of the list, we want to append to the end.